src/vdaf/poplar1.rs

```diff
@@ -822,41 +833,23 @@ mod tests {
 
     #[test]
     fn test_poplar1() {
-        let vdaf: Poplar1<ToyIdpf<Field128>> = Poplar1::new(Suite::Blake3);
-        assert_eq!(vdaf.num_aggregators(), 2);
+        const INPUT_LEN: usize = 16;
```

Comment:
to cut down on the runtime of the unit tests, we ought to cut this down to 8 bits. We should be able to modify the tests cases accordingly and still have the same coverage.